installer/bundle/bundle_skel.sh

```diff
@@ -1551,11 +1551,12 @@ you should first purge the existing installation and then install using the --sk
 
         # Update OMS Agent
         shouldInstall_omsagent
+        temp_status=$?
         rm -f "$OMS_CONSISTENCY_INVOKER" > /dev/null 2>&1
```

Comment:
couldn't we just move this before `shouldInstall_omsagent` and have no other change necessary?

Reply:
Moving rm would not make any difference. This change is to explicitly  say that the return value of shouldinstall is used as a parameter in the update function. Because it was missing, it resulted in this bug.

Reply:
but imagine the code is like this

```
# Update OMS agent
rm -f "$OMS_CONSISTENCY_INVOKER" > /dev/null 2>&1
shouldInstall_omsagent
pkg_upd $OMS_PKG omsagent $?
```

How is `pkg_update` not using the return value of `shouldInstall_omsagent`?